geode-core/src/main/java/org/apache/geode/internal/InternalDataSerializer.java

```diff
@@ -2273,8 +2282,12 @@ public void write(int b) throws IOException {
    * @param out the output stream.
    */
   public static void invokeToData(Object ds, DataOutput out) throws IOException {
-    boolean isDSFID = ds instanceof DataSerializableFixedID;
     try {
+      boolean isDSFID = ds instanceof DataSerializableFixedID;
```

Comment:
this variable is used only once—I suggest you inline the expr

Reply:
okay, will do

Reply:
done